src/haxelib/client/Main.hx

```diff
@@ -68,10 +70,15 @@ class ProgressOut extends haxe.io.Output {
 
 	function report(n) {
 		cur += n;
+		curMB = cur / 1000000.0;
+		curMB = Math.round( curMB * Math.pow(10, 2) ) / Math.pow(10, 2); // 12.34 precision.
```

Comment:
Why not `100`? I don't think it's more readable this way, and it's marginally slower.